apps/doc-gen/tsconfig.json

```diff
@@ -2,6 +2,8 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "jsx": "react-jsx",
+    "target": "ES2022",
```

Comment:
so only this plugin needs 2022? It's extended from base.

Reply:
We could make the base to target ES2022, but this would have to be agreed upon by us here.